charts/fpl-case-service/values.preview.template.yaml

```diff
@@ -5,6 +5,7 @@ tags:
 java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  readinessDelay: 60
```

Comment:
Shouldn't that be done for liveness probe? Readiness does not restart pod, does it? Is't readiness used just to dec idea whether traffic should go to the pod or not?

Reply:
Of course you are eright that should be liveness!